tests/kernel/tuned.pm

```diff
@@ -0,0 +1,67 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Regression test for tuned daemon
+# Maintainer: Petr Cervinka <pcervinka@suse.com>
+# Tags: https://jira.suse.com/browse/SLE-6514
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub tuned_set_profile {
+    my $tuned_profile = shift;
+    assert_script_run "tuned-adm profile $tuned_profile";
+    validate_script_output 'tuned-adm active', sub { m/${tuned_profile}/ };
+}
+
+sub run {
+    my $self      = shift;
+    my $tuned_log = '/var/log/tuned/tuned.log';
+    # Already known errors with bug reference
+    my %known_errors = (
+        bsc_1148789 => 'Executing cpupower error: Error setting perf-bias value on CPU'
+    );
+    select_console 'root-console';
```

Comment:
just out of curiosity - any reason why not to run this test with VIRTIO_CONSOLE=1 ?

Reply:
> I am not sure about the soft-failure. Showing 'red' in the tests actually is a sign of that something is wrong, in this case an already reported bug... The bug can be added to the job as comment later on as a reason that it fails due to bug X.

tuned.log can have many errors and commenting on the job level wouldn't work well in this case

Reply:
> just out of curiosity - any reason why not to run this test with VIRTIO_CONSOLE=1 ?

I tried and I had issues with select_serial_terminal  on spvm backend and I put it on my never ending todo to have a look.

Reply:
make sense